dev-qt/qtbase/qtbase-6.7.2-r1.ebuild

```diff
@@ -13,7 +13,7 @@ fi
 
 declare -A QT6_IUSE=(
 	[global]="+ssl +udev zstd"
-	[core]="icu"
+	[core]="icu journald syslog systemd"
```

Comment:
>I've added systemd to IUSE to fix the CI issue. Please tell me if there is a better solution.

I didn't see what error CI gave before you changed that, but I assume it's musl profile, so just need to mask USE=journald on in `profiles/features/musl/package.use.mask`

Could just add it to sam's mask to keep them together, aka:
```
# Sam James <sam@gentoo.org> (2022-10-03)
# Needs systemd which is masked on musl
```

Reply:
(if there's more issues, feel free to leave CI broken rather than keep the workaround in, I can always sort that out on merge)

Reply:
Yes, that is indeed the issue. I've added the mask.